docs/src/modules/utils/helpers.ts

```diff
@@ -226,13 +226,16 @@ export function getDependencies(
  * @return The cookie value
  */
 export function getCookie(name: string): string | undefined {
-  if (typeof document !== 'undefined') {
-    const value = `; ${document.cookie}`;
-    const parts = value.split(`; ${name}=`);
-    if (parts.length === 2) {
-      return parts[1].split(';').shift();
-    }
+  if (typeof document === 'undefined') {
+    throw new Error('Your logic is not SSR mode safe');
```

Comment:
These types of error messages are just not helpful. Please consider the people encountering them.
```suggestion
    throw new Error('getCookie() is not supported on the server. Fallback to a different value when rendering on the server.');
```